component-library/src/app/components/side-nav/side-nav.component.ts

```diff
@@ -61,7 +67,9 @@ export class SideNavComponent implements OnInit, AfterViewChecked {
       document.documentElement.clientHeight;
     const sideNavTitles: NodeListOf<HTMLHeadElement> =
       document.querySelectorAll('app-title-slug-url h1, app-title-slug-url h2');
-    const sideNavLinks = document.querySelectorAll('ircc-cl-lib-nav-item a');
+    // Query all side nav anchor tags from current elementRef
+    const sideNavLinks: NodeListOf<HTMLAnchorElement> =
+      this.el.nativeElement.querySelectorAll('ircc-cl-lib-nav-item a');
```

Comment:
@alexandreGrenierIRCC This can't be done by ViewChildren as I need to query into the nav-item component.